pkg/imgpkg/lockfiles/lock_files.go

```diff
@@ -1,25 +1,27 @@
 // Copyright 2020 VMware, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-package cmd
+package lockfiles
 
 import (
 	"fmt"
 	"io/ioutil"
 
-	regname "github.com/google/go-containerregistry/pkg/name"
 	ctlimg "github.com/k14s/imgpkg/pkg/imgpkg/image"
 
-	"github.com/google/go-containerregistry/pkg/name"
+	regname "github.com/google/go-containerregistry/pkg/name"
 	"gopkg.in/yaml.v2"
 )
 
 const (
-	ImageLockKind  string = "ImageLock"
+	ImagesLockKind string = "ImagesLock"
```

Comment:
I was expecting this change should have broken some tests.
We need to make sure that at least some of our tests do not depend on these constants as this can hurt us in the future.

Reply:
Removed and replaced with comments on why not using consts.